### DISCUSSION TITLE
【广告】在外环，我们都用金坷垃

### DISCUSSION THREAD

**Discussion by chen-SJc** [share121/inter-knot#5090] [其他]
![241f95cad1c8a7868a0312246d09c93d70cf50ea](https://github.com/user-attachments/assets/afe47cab-6c6a-473a-a0fe-e72170561ef7)
空洞农业不发达，必须要有 **[金坷垃](https://www.bilibili.com/video/BV1Rs411R7Hi/?spm_id_from=333.337.search-card.all.click&vd_source=c49177b0cedfd8bbc94f696a6e2e5a7c)**
外环资源太缺乏，必须要有 **[金坷垃](https://www.bilibili.com/video/BV1Rs411R7Hi/?spm_id_from=333.337.search-card.all.click&vd_source=c49177b0cedfd8bbc94f696a6e2e5a7c)**
***
# 你家麦地两米以下藏着啥？
**[外环地多施肥多，为什么小麦产量却远低于预期？](https://www.bilibili.com/video/BV1Yu411e78i?p=5&vd_source=c49177b0cedfd8bbc94f696a6e2e5a7c)**
因为外环的农民不知道，他们的麦地2米下藏着什么
**地面2米以下的土壤样本** 
经精密测量，它含有60%的氮，90%的高磷高钾。一个土块所含的氮磷钾，相当于**1公斤**复合肥的含量。
***
# 金坷垃好处都有啥？
**肥料掺了[金坷垃](https://www.bilibili.com/video/BV1Rs411R7Hi/?spm_id_from=333.337.search-card.all.click&vd_source=c49177b0cedfd8bbc94f696a6e2e5a7c)，不流失，不蒸发，零浪费** 
**肥料掺了[金坷垃](https://www.bilibili.com/video/BV1Rs411R7Hi/?spm_id_from=333.337.search-card.all.click&vd_source=c49177b0cedfd8bbc94f696a6e2e5a7c)，能吸收两米下的氮磷钾**
**世界肥料都涨价，肥料掺了肥料掺了[金坷垃](https://www.bilibili.com/video/BV1Rs411R7Hi/?spm_id_from=333.337.search-card.all.click&vd_source=c49177b0cedfd8bbc94f696a6e2e5a7c)，能吸收两米下的氮磷钾**
**用了[金坷垃](https://www.bilibili.com/video/BV1Rs411R7Hi/?spm_id_from=333.337.search-card.all.click&vd_source=c49177b0cedfd8bbc94f696a6e2e5a7c)，小麦亩产一千八，拉面店的食材再也不会短缺啦**
# [New Eridu·SHENGDIYAGE](https://www.bilibili.com/video/BV1Yu411e78i?p=8&vd_source=c49177b0cedfd8bbc94f696a6e2e5a7c)

**Comment by BJ-cmyk** [comment]
金坷垃很好用，给尼尼微吃直接变成烈度99了💀

**Reply by cuijiaqi0406** [reply]
bro以为尼尼微能当小麦吃💀

**Comment by Liruijia666** [comment]
牛逼

**Comment by FairyDeer2333** [comment]
好吃吗😋

**Comment by LiuLiBHZ** [comment]
听说干嚼金坷垃可以提升以太抗性，我朋友最近要考调查员执照我帮他问问。
